### ISSUE TITLE
Table cells not repeating as they should on next page when table does not fit in one page

### ISSUE THREAD

**Issue by mctsnt** [QuestPDF/QuestPDF#320]
**Describe the bug**
Issue noticed in v2022.8.0
I have a table I populate dynamically, the first column has a Row span of all the rows the table will eventually have, and I expect when the rows are too many and the table is cut into two pages for the first column to be repeated in both pages, however it is only shown on 1 page.
I tried to upgrade to v2022.8.1 but then the document would not even render and I got the layout error but my document is only 16 pages and renders fine in 2022.8.0

**To Reproduce**

```
container.Table(table =>
            {
                table.ColumnsDefinition(columns =>
                {
                    columns.RelativeColumn();
                    columns.RelativeColumn();
                    columns.RelativeColumn();
                    columns.RelativeColumn();
                });

                    foreach (var device_incident in incidents_model)
                    {
                        var span = Convert.ToUInt16(device_incident.Key.Total + 1);
                        table.Cell().ColumnSpan(4).Element((container) => BlockOverview(container, Color_Linked_View_Device_Headers)).Text(device_incident.Key.Total.ToString());
                        table.Cell().RowSpan(span).Element((container) => BlockOverview(container, Color_Linked_View_Device_Headers)).Text(device_incident.Key.Name);
                        table.Cell().Element((container) => BlockOverview(container, Color_Linked_View_Headers)).Text("Incident Id");
                        table.Cell().ColumnSpan(2).Element((container) => BlockOverview(container, Color_Linked_View_Headers)).Text("Incident title");
                        foreach (var incident in device_incident.Value)
                        {
                            table.Cell().Element((container) => BlockOverview(container, Color_White)).Text(incident.IncidentId);
                            table.Cell().ColumnSpan(2).Element((container) => BlockOverview(container, Color_White)).Text(incident.IncidentTitle);
                        }
                    }
            });
static IContainer BlockOverview(IContainer container, string color, string bColor = Colors.White)
    {
        return container
            .Border(1)
            .BorderColor(bColor)
            .Background(color)
            .MinWidth(20)
            .MinHeight(20)
            .AlignMiddle()
            .AlignCenter()
            ;
    }


```
**Screenshots**
page 1:
![image](https://user-images.githubusercontent.com/59544132/185639731-ba43523e-2a03-4371-aa45-08fa095fb504.png)
page 2: (where the table continues)
![image](https://user-images.githubusercontent.com/59544132/185639850-9cb0db41-cd90-440a-a141-7c914a94bddd.png)

**Expected behavior**
I expect the view to be something like this
page 1:
![image](https://user-images.githubusercontent.com/59544132/185640030-d75dd01a-984b-4344-a7e3-a6f2b9b083e3.png)
page 2: 
![image](https://user-images.githubusercontent.com/59544132/185640115-07c32a77-6b51-4e3e-88e0-70487707594c.png)

**Comment by MarcinZiabek** [comment]
I will investigate the table rendering algorithm to check if everything is working as designed. I am not sure where exactly I will be able to perform any updates. For now, you can always use simpler structures to accomplish similar results, e.g. Column and Row elements.

**Comment by mctsnt** [comment]
Ok I will try something like that in the meantime, thank you for your response!

**Comment by MarcinZiabek** [comment]
I have fixed the issue. The release with the fix will be available soon.

**Comment by MarcinZiabek** [comment]
This problem has been fixed as part of the 2022.11 release. Closing
